### PR TITLE
[FIX] Also annotate first file when merging cXML

### PR DESCRIPTION
**Pull request by jpfeuffer**

### CODE REVIEW

**Comment by jpfeuffer** [comment]
build pyopenms

**Comment by jpfeuffer** [comment]
build pyopenms

**Comment by jpfeuffer** [comment]
Pyopenms Trigger works, too.

**Comment by timosachsenberg** [comment]
so the magic words to trigger are biudl ypopenms?

File: src/topp/FileMerger.cpp [hunk 1]
```diff
@@ -245,8 +245,18 @@ class TOPPFileMerger :
     {
       ConsensusMap out;
       ConsensusXMLFile fh;
+      // load the metadata from the first file
       fh.load(file_list[0], out);
-      // skip first file
+      // but annotate the origins
+      if (annotate_file_origin)
+      {
+        for (ConsensusMap::iterator it = out.begin(); it != out.end(); ++it)
```
Comment (timosachsenberg) [inline src/topp/FileMerger.cpp]: We could think about using for_each and a lambda in the future.
in my view its even easier to read

Comment (jpfeuffer) [inline src/topp/FileMerger.cpp]: true.. was copy pasta

**Comment by jpfeuffer** [comment]
Haha yes :) up for discussion. can be a regex, too. Have to fix the builds a little bit though. Does not run through yet.

**Comment by hroest** [comment]
very nice, I like it a lot! Thanks @jpfeuffer seems like there is an issue with the extension checker, maybe a missing argument?

**Pull request merged**